native/jni/src/main/java/com/intel/analytics/bigdl/mkl/MKL.java

```diff
@@ -17,13 +17,24 @@
 
     static {
         try {
-            tmpFile = extract("libiomp5.so");
-            System.load(tmpFile.getAbsolutePath());
-            tmpFile.delete(); // delete so temp file after loaded
-            tmpFile = extract("libjmkl.so");
-            System.load(tmpFile.getAbsolutePath());
-            tmpFile.delete(); // delete so temp file after loaded
-            isLoaded = true;
+            if (System.getProperty("os.name").toLowerCase().contains("mac")) {
```

Comment:
too much duplicate code here?  I think you just need set the file name based on os.name